integrationtests/as-integration-client/pom.xml

```diff
@@ -72,6 +72,18 @@
             <groupId>org.infinispan</groupId>
             <artifactId>infinispan-client-hotrod</artifactId>
         </dependency>
+        <dependency>
```

Comment:
@slaskawi These are already in the remote client uberjar. But are they tested? It seems ```RemoteAllTest``` is all we have atm.